templates/profiles/role_detail.html

```diff
@@ -0,0 +1,60 @@
+{% extends 'base.html' %}
+{% block title %}
+    #{{ object.id }} | Role
+{% endblock %}
+{% load render_table from django_tables2 %}
+{% load static %}
+{% block header_button %}
+    {% if request.user.is_superuser %}
+        <div class="float-right">
+            <a href="{% url 'profiles:role_edit' object.id %}"
+               class="btn btn-primary">
+                <i class="fas fa-pencil-alt"></i>
+            </a>
+            <a href="{% url 'profiles:role_delete' object.id %}"
+               class="btn btn-danger">
+                <i class="fas fa-trash"></i>
+            </a>
+        </div>
+    {% endif %}
+{% endblock %}
+
+{% block main %}
+    <div class="container-fluid">
+        <div class="row">
+            <section class="col-lg-3">
+                <div class="card card-sucess">
```

Comment:
TODO (maybe in a next PR). Remove the usage of a card here.
![image](https://github.com/HewlettPackard/squest/assets/2573405/a2b4c719-d3a1-4b3f-aa3e-8a8d6fb8eeb6)